src/components/thumbnail/simple-document-item.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { IDocumentMetadata } from "../../../functions/src/shared";
+import { useStores } from "../../hooks/use-stores";
+
+import "./simple-document-item.scss";
+
+interface IProps {
+  document: IDocumentMetadata;
+  investigationOrdinal: number;
+  problemOrdinal: number;
+  onSelectDocument: (document: IDocumentMetadata) => void;
+}
+
+export const SimpleDocumentItem = ({ document, investigationOrdinal, onSelectDocument, problemOrdinal }: IProps) => {
+  const { class: classStore, unit } = useStores();
+  const { uid } = document;
+  const userName = classStore.getUserById(uid)?.displayName;
+  const investigations = unit.investigations;
+  const investigation = investigations[investigationOrdinal];
+  const problem = investigation?.problems[problemOrdinal - 1];
+  const title = document.title ? `${userName}: ${document.title}` : `${userName}: ${problem?.title ?? "unknown title"}`;
```

Comment:
If possible, this should be moved to a common place since I think basically the same logic is used in multiple places now.

Reply:
Is that true? Are you thinking of the title values when you click on a box for a document outside the current problem to view it? At least for now, there we're just using "Unknown Problem" which comes from `getDocumentDisplayTitle` in document-utils.ts. Were you think that function could be modified to handle putting together at least some part of the `title` value here? Or is there some other place where we're already looking up titles for things outside the current problem?

Reply:
Yeah I was thinking of how this kind of duplicates `getDocumentDisplayTitle`.  
Here are some more places I've found:
- `GroupTitlebar` in student-group-view
- `useDocumentCaption` this uses getDocumentDisplayTitle

So perhaps the best thing would be to see if getDocumentDisplayTitle could be used here.

Reply:
For the record, we'll be looking into doing this as part of a separate story.